src/Microsoft.IdentityModel.Protocols.WsFederation/WsFederationMetadataSerializer.cs

```diff
@@ -271,5 +274,93 @@ internal static string Trim(string stringToTrim)
             char[] charsToTrim = { ' ', '\n' };
             return stringToTrim.Trim(charsToTrim);
         }
+
+#endregion
+
+#region Write Metadata
+
+        /// <summary>
+        /// Write the content in configuration into writer.
+        /// </summary>
+        /// <param name="writer">The <see cref="XmlWriter"/> used to write the configuration content.</param>
+        /// <param name="configuration">The <see cref="WsFederationConfiguration"/> provided.</param>
+        /// <exception cref="ArgumentNullException">if writer or configuration parameter is missing.</exception>
```

Comment:
please use  <paramref ...>
see: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/blob/dev/src/Microsoft.IdentityModel.Tokens.Saml/Saml2/Saml2SecurityToken.cs#L43